lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -13,6 +13,7 @@
                 errorMessage: gettext('An error occurred. Try again.'),
                 alreadyMemberMessage: gettext('You already belong to another team.'),
                 teamFullMessage: gettext('This team is full.'),
+                notJoinInstructorManagedTeam: gettext('Cannot join instructor managed team'),
```

Comment:
Good call on having some user-friendly messaging here explaining why they can't join the team.

@dabdul-hathi --
* what should the exact text here be?
* should there be similar messaging explaining why you can't _leave_ an instructor-managed team?